extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -35,6 +35,12 @@ const VISIBILITY_ID_PROP = '__AMP_VIS_ID';
 /** @type {number} */
 let visibilityIdCounter = 1;
 
+/** @enum {string} */
+export const RelativePosition = {
```

Comment:
nit: can we reuse the concept from 
https://github.com/ampproject/amphtml/blob/efba3b226fa1f6d49548cfd4c7331ce070c1af80/src/layout-rect.js#L70-L74

Reply:
Done.